examples/tutorials/advanced/websockets-example-MNIST/run_websocket_client_worker.py

```diff
@@ -1,7 +1,7 @@
 from multiprocessing import Process
 
 import syft as sy
-from syft.workers import WebsocketServerWorker
+from syft.workers import WebsocketClientWorker
```

Comment:
I do not like this name change. Traditionally the client is the one who calls commands. I would prefer if we moved away from the terms client and server regardless as they seem to add confusion. That being said I would recommend that we spend more time brainstorming names before we change them

Reply:
@robert-wagner Thanks for your feedback. 
What names do you propose? And whose opinion should be heard?